cost/billing_center_cost_anomaly/billing_center_cost_anomaly.pt

```diff
@@ -0,0 +1,294 @@
+name "Billing Center Cost Anomalies"
+rs_pt_ver 20180301
+type "policy"
+short_description "Analyze all Billing Centers for a specified number of days and raise an incident if the percentage of spend (compared to the previous period of the same number of days) has surpassed the defined threshold. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/cost_anomaly/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
Link needs to be updated to `[README](https://github.com/rightscale/policy_templates/tree/master/cost/billing_center_cost_anomaly/)`